src/i18n/strings/en_EN.json

```diff
@@ -3091,6 +3091,8 @@
     "Your message wasn't sent because this homeserver has hit its Monthly Active User Limit. Please <a>contact your service administrator</a> to continue using the service.": "Your message wasn't sent because this homeserver has hit its Monthly Active User Limit. Please <a>contact your service administrator</a> to continue using the service.",
     "Your message wasn't sent because this homeserver has been blocked by its administrator. Please <a>contact your service administrator</a> to continue using the service.": "Your message wasn't sent because this homeserver has been blocked by its administrator. Please <a>contact your service administrator</a> to continue using the service.",
     "Your message wasn't sent because this homeserver has exceeded a resource limit. Please <a>contact your service administrator</a> to continue using the service.": "Your message wasn't sent because this homeserver has exceeded a resource limit. Please <a>contact your service administrator</a> to continue using the service.",
+    "Your message wasn't sent. Your account is suspended: you cannot send messages, invites or join new rooms. To learn more, visit <detailsLink>this page</detailsLink>.": "Your message wasn't sent. Your account is suspended: you cannot send messages, invites or join new rooms. To learn more, visit <detailsLink>this page</detailsLink>.",
+    "Your message wasn't sent. Your account is suspended: you cannot send messages, invites or join new rooms. Please contact the administrator of %(homeserverDomain)s.": "Your message wasn't sent. Your account is suspended: you cannot send messages, invites or join new rooms. Please contact the administrator of %(homeserverDomain)s.",
```

Comment:
as already mentioned by @t3chguy, the status bar itself looks broken with this change, so we'd need to fix that before landing this PR.

The options for that are:
1. Wait for someone on the web team to get to it in the backlog (slow, and likely way too slow for your timelines).
2. Get your hands dirty and learn some CSS 😇 
3. Find someone who is able to help with writing that CSS and have them contribute ideas/guidance/code to this PR

Personally, I'd encourage option 2, particularly if you're expecting to be in the Element Web area for future changes.

Reply:
As already responded to @t3chguy, the status bar itself is already broken prior to this change.

I would really, really, really like to avoid scope creep. Can we please postpone this to a followup issue?

Reply:
Even if it was broken before, we can't reasonably land something which has a near-100% chance of causing the bug. It just doesn't line up with our code quality requirements, sorry.